src/components/FavouriteStay/FavoriteStay.jsx

```diff
@@ -0,0 +1,60 @@
+import React from "react";
+import StarRating from "../StarRating/Starrating";
+import "./FavoriteStay.css"
+import Left from "../../assets/fav-badge-left.png";
+import Right from "../../assets/fav-badge-right.png";
+
+/* rating vlaue to be obtained from DB */
+const FavoriteStay = ({ rating = 5, reviews = 71 }) => {
+  return (
+    <div className="fav-stay-container">
+      <div className="fav-stay-grid-container">
+        {/* badge */}
+        <div></div>
+        <div className="host-badge-container">
+          <div className="host-badge-left">
+            <img className="left" src={Left} 
+            alt="fav-badge-left" />
+          </div>
+
+          <div className="host-badge-text">Guest{<br></br>}favorite</div>
+
+          <div className="host-badge-right">
+          <img className="right" src={Right}
+            alt="fav-badge-right" />
+          </div>
+        </div>
+
+        <div></div>
```

Comment:
This <div> tag seems redundant as there's nothing between the tags. There are few similar tags in this file.

Reply:
they are actually useful, amusing them in the grid-template-columns, probably there is a better way to do it